Branch-SDK/src/main/java/io/branch/referral/network/BranchRemoteInterfaceUrlConnection.java

```diff
@@ -116,6 +117,7 @@ private BranchResponse doRestfulGet(String url, int retryNumber) throws BranchRe
 
     private BranchResponse doRestfulPost(String url, JSONObject payload, int retryNumber) throws BranchRemoteException {
         HttpsURLConnection connection = null;
+        String requestId = null;
```

Comment:
Variable declaration can be pulled into the try block (where value is assigned) for brevity.

Reply:
Fair